src/views/MetocDataView.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="display-container">
+    <SplashScreen />
```

Comment:
We now have the splashscreen only here, maybe we can skip the base route and always start here as well when we route to the viewer. Otherwise it should also be added in the start layout.